weblate/vcs/tests/test_vcs.py

```diff
@@ -164,7 +165,12 @@ def test_weblate_repo_init(self):
     def tearDown(self):
         self.remove_temp()
 
-    def add_remote_commit(self, conflict=False, rename=False):
+    def add_remote_commit(
+        self,
+        conflict=False,
+        rename=False,
+        binary: Optional[bool] = False,
```

Comment:
```suggestion
        binary: bool = False,
```

This is not optional – that means that it accepts None as well.